acceptance/setup/git/pre-suite/010_TestSetup.rb

```diff
@@ -1,85 +1,76 @@
 test_name "Install packages and repositories on target machines..." do
-require 'beaker/dsl/install_utils'
-extend Beaker::DSL::InstallUtils
+  require 'beaker/dsl/install_utils'
+  extend Beaker::DSL::InstallUtils
 
-  SourcePath  = Beaker::DSL::InstallUtils::SourcePath
-  GitURI      = Beaker::DSL::InstallUtils::GitURI
   GitHubSig   = Beaker::DSL::InstallUtils::GitHubSig
 
-  tmp_repositories = []
-  options[:install].each do |uri|
-    raise(ArgumentError, "Missing GitURI argument. URI is nil.") if uri.nil?
-    raise(ArgumentError, "#{uri} is not recognized.") unless(uri =~ GitURI)
-    tmp_repositories << extract_repo_info_from(uri)
-  end
+  gem_source = ENV["GEM_SOURCE"] || "https://rubygems.org"
+  sha = ENV['SHA'] || `git rev-parse HEAD`.chomp
+  repository = extract_repo_info_from(build_git_url("puppet"))
```

Comment:
I'm not a huge fan of hard coding `puppet` here, especially since this utilities will eventually be moved into `beaker-puppet`

So, above, we're using the options file to define install targets like `puppet`. Previously before doing git testing, we'd parse those install targets into git urls:
```
   final_options[:install].map! do |install|		
      raise(ArgumentError, "Missing Git URL within options hash. Install URL is nil.") if install.nil?		
      if md = /^(\w+)#([\d.\w]+)$/.match(install)		
        project, project_sha = md.captures		
        "#{build_giturl(project)}##{project_sha}"		
      elsif md = /^(\w+)$/.match(install)		
        project = md[1]		
        "#{build_giturl(project)}##{sha}"		
      else		
        install		
      end		
    end
```
I think it would make sense to do something similar here as well